src/components/20-molecules/datepicker/utils/date.js

```diff
@@ -46,7 +46,7 @@ const dayToCell = (day, date, selectedDate, today, allowedYears) => ({
   inactive: allowedYears.indexOf(date.getFullYear()) < 0,
 });
 
-const getMonthMatrix = (date, allowedYears = [], selectedDate) => {
+const getMonthMatrix = (date, selectedDate, allowedYears = []) => {
```

Comment:
I get what you're doing here: defaulty args should come last. I guess this is code that was extended/refined over time.
In general, such refactoring could obscure the original importance ordering of arguments: left-most is most important, second leftmost is ....
Also, it could affect readability when instantiated/called. So, be careful with this transformation in general.